package.json

```diff
@@ -87,19 +91,30 @@
     "lerna": "2.0.0-rc.4",
     "load-json-file": "^2.0.0",
     "nightwatch": "^0.9.16",
+    "node-sass": "^4.6.0",
+    "postcss-custom-properties": "^6.2.0",
+    "postcss-loader": "^2.0.8",
+    "postcss-rtl": "^1.1.3",
+    "prop-types": "^15.6.0",
     "react": "^15.6.1",
     "react-dom": "^15.6.1",
     "react-router": "^3.0.5",
     "react-router-dom": "^4.2.2",
     "react-test-renderer": "^15.5.4",
     "rimraf": "^2.6.1",
+    "sass-loader": "^6.0.6",
     "selenium-server-standalone-jar": "^3.4.0",
     "shelljs": "^0.7.7",
+    "style-loader": "^0.19.0",
     "stylelint": "~8.0.0",
     "stylelint-config-sass-guidelines": "^3.0.0",
     "stylelint-scss": "1.5.1",
     "stylelint-suitcss": "^1.0.0",
+    "terra-base": "^2.6.1",
     "terra-i18n": "^1.4.0",
+    "terra-i18n-plugin": "file:../terra-core/packages/terra-i18n-plugin",
```

Comment:
just for local dev, after this PR is merged we can update it with the new version